saleor/graphql/shop/mutations.py

```diff
@@ -390,7 +397,12 @@ class Arguments:
         )
 
     class Meta:
-        description = "Update shop order settings."
+        description = (
+            "Update shop order settings across all channels"
+            "\nReturns `orderSettings` for the first `channel` "
+            f"in alphabetical order.{DEPRECATED_IN_3X_MUTATION}"
```

Comment:
See other usages of `DEPRECATED_IN_3X_MUTATION`. We should pass it as `deprecation_reason` in the field definition like this:
```
order_settings_update = OrderSettingsUpdate.Field(
    deprecation_reason=DEPRECATED_IN_3X_MUTATION
)
```